<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-CNPB3TNK" failures="1" tests="7" name="com.spree.Create_Multiple_Addresses_Using_TestData" time="3.974" errors="0" timestamp="2024-03-14T13:16:32 IST" skipped="6">
  <testcase classname="com.spree.Create_Multiple_Addresses_Using_TestData" name="pre_condition" time="3.974">
    <failure message="expected [301] but found [204]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [301] but found [204]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at com.spree.BaseClass.deleteAllAddresses(BaseClass.java:61)
at com.spree.Create_Multiple_Addresses_Using_TestData.pre_condition(Create_Multiple_Addresses_Using_TestData.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- pre_condition -->
  <system-out/>
  <testcase classname="com.spree.Create_Multiple_Addresses_Using_TestData" name="addAddress" time="0.000">
    <skipped/>
  </testcase> <!-- addAddress -->
  <system-out/>
  <testcase classname="com.spree.Create_Multiple_Addresses_Using_TestData" name="addAddress" time="0.000">
    <skipped/>
  </testcase> <!-- addAddress -->
  <system-out/>
  <testcase classname="com.spree.Create_Multiple_Addresses_Using_TestData" name="addAddress" time="0.000">
    <skipped/>
  </testcase> <!-- addAddress -->
  <system-out/>
  <testcase classname="com.spree.Create_Multiple_Addresses_Using_TestData" name="addAddress" time="0.000">
    <skipped/>
  </testcase> <!-- addAddress -->
  <system-out/>
  <testcase classname="com.spree.Create_Multiple_Addresses_Using_TestData" name="addAddress" time="0.000">
    <skipped/>
  </testcase> <!-- addAddress -->
  <system-out/>
  <testcase classname="com.spree.Create_Multiple_Addresses_Using_TestData" name="countAddress" time="0.000">
    <skipped/>
  </testcase> <!-- countAddress -->
  <system-out/>
</testsuite> <!-- com.spree.Create_Multiple_Addresses_Using_TestData -->
